crates/typst-cli/src/world.rs

```diff
@@ -384,6 +395,15 @@ fn decode_utf8(buf: &[u8]) -> FileResult<&str> {
     Ok(std::str::from_utf8(buf.strip_prefix(b"\xef\xbb\xbf").unwrap_or(buf))?)
 }
 
+/// The current date and time.
+enum Today {
```

Comment:
This is an awkward name for a struct that stores date *and* time. At first I thought this PR also changed the time granularity to only count whole days. What's wrong with `Now`?